cmd/vcluster/cmd/start.go

```diff
@@ -124,6 +129,9 @@ func NewStartCommand() *cobra.Command {
 
 	cmd.Flags().StringVar(&options.DefaultImageRegistry, "default-image-registry", "", "This address will be prepended to all deployed system images by vcluster")
 
+	cmd.Flags().BoolVar(&options.EnforcePodSecurityStandard, "enforce-pod-security-standard", false, "If enabled and --pod-security-standard is set then the virtual cluster will enforce specified pod security standard")
+	cmd.Flags().StringVar(&options.PodSecurityStandard, "pod-security-standard", "privileged", "This can be set to privileged, baseline, restricted (defaults to privileged) and vcluster would make sure during translation that these policies are enforced.")
```

Comment:
Merge this into a single flag called `enforce-pod-security-standard` that is empty by default, but as soon as its defined its enforced at the same time